.zshrc

```diff
@@ -114,27 +114,41 @@ test -e "${HOME}/.iterm2_shell_integration.zsh" && source "${HOME}/.iterm2_shell
 function iterm2_print_user_vars() {
   iterm2_set_user_var gitDiff $(is_git_branch_dirty)
   iterm2_set_user_var humpDay $(is_it_wednesday)
-  iterm2_set_user_var gitDir $(getGitProjectDir)
+  iterm2_set_user_var gitDir $(showGitProject)
 }
 
 function is_git_branch_dirty {
   [[ $(git diff --shortstat 2> /dev/null | tail -n1) != "" ]] && echo "⚡⚡"
 }
 
 function is_it_wednesday {
-  if [[ $(date +%A) = "Wednesday" ]]
-    then
+  local date=$(date +%A)
+
+  if [[ "$date" = "Wednesday" ]] then
      echo "🐪" # Camel Prompt
-    else
+  elif [[ "$date" = "Tuesday" ]] then
+    echo "🌮🌮"
+  elif [[ "$date" = "Friday" ]] then
+    echo "🍕"
+  elif [[ "$date" = "Saturday" ]] then
+    echo "🛌"
+  else
```

Comment:
TODO: refactor this function and make it more generic. Also switch case it for each day?